### PR TITLE
GraphNG: sort ascending if the values appear reversed

### PR DESCRIPTION
**Pull request by ryantxu**
The new graph panel will draw things if they came out of order, but all interactions assume ascending order.

This PR does a quick check (last > first) and may automatically apply a sort to the data.

### CODE REVIEW

**Comment by ryantxu** [comment]
@joanlopez -- this should help when data is streamed reverse chronological!

File: packages/grafana-ui/src/components/GraphNG/utils.ts [hunk 1]
```diff
@@ -58,26 +59,30 @@ export function alignDataFrames(frames: DataFrame[], fields?: XYFieldMatchers):
...
     }
 
-    if (dims.x.length > 1) {
-      throw new Error('Only a single x field is supported');
+    // Quick check that x is ascending order
+    if (!isLikelyAscendingVector(dims.x.values)) {
+      const xIndex = frame.fields.indexOf(dims.x);
+      frame = sortDataFrame(frame, xIndex);
+      dims = mapDimesions(fields, frame, frames);
+      console.log('SORTed!!!?');
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/GraphNG/utils.ts:74]: nit: please remove

**Review by hugohaggmark — APPROVED** [review]
LGTM! 🌟

**Comment by joanlopez** [comment]
> @joanlopez -- this should help when data is streamed reverse chronological!

Amazing, thanks for pinging me @ryantxu 👏 🙏

**Pull request merged**